bugbug/tools/code_review.py

```diff
@@ -62,16 +62,29 @@ class HunkNotInPatchError(ModelResultError):
 
 {patch}"""
 
-PROMPT_TEMPLATE_REVIEW = """You will be given a task for generate a code review for the patch below. Use the following steps to solve it.
-
-{patch}
-
+PROMPT_TEMPLATE_REVIEW = """You will be given a task for generate a code review for the patch below. Use the following steps to solve it:
 1. Understand the changes done in the patch by reasoning about the summarization as previously reported.
 2. Identify possible code snippets that might result in possible bugs, major readability regressions, and similar concerns.
-3. Reason about each identified problem to make sure they are valid. Have in mind, your review must be consistent with the source code in Firefox. As valid comments, consider the examples below:
-{comment_examples}
+3. Reason about each identified problem to make sure they are valid. Have in mind, your review must be consistent with the source code in Firefox.
```

Comment:
Note: we should make this customizable so other users can replace "Firefox" with something else. Let's open a follow-up for that

Reply:
I filed a follow-up issue: https://github.com/mozilla/bugbug/issues/4319.